config/peerpods/podvm/libvirt-podvm-image-handler.sh

```diff
@@ -21,23 +21,73 @@ function verify_vars() {
 
     [[ -z "${ORG_ID}" ]] && error_exit "ORG_ID is not set"
     [[ -z "${ACTIVATION_KEY}" ]] && error_exit "ACTIVATION_KEY is not set"
-    [[ -z "${BASE_OS_VERSION}" ]] && error_exit "BASE_OS_VERSION is not set"
 
-    [[ -z "${PODVM_DISTRO}" ]] && error_exit "PODVM_DISTRO is not set"
+    if [[ "${IMAGE_TYPE}" == "operator-built" ]]; then
```

Comment:
Can you also update podvm-handling.md explaining the two image types ie operator-built and pre-built

Reply:
incorporated..